### PR TITLE
listDirAtPath: expose members

### PR DESCRIPTION
**Pull request by maurycy**

### CODE REVIEW

File: core/space/domain/domain.go [hunk 1]
```diff
@@ -154,5 +155,5 @@ type SharedDirEntry struct {
 	DbID   string
 	Bucket string
 	FileInfo
-	Members []Member
+	Members []Member // XXX: it is duplicated from FileInfo
```
Comment (jsonsivar) [inline core/space/domain/domain.go:158]: Okay maybe we can remove this then since it's included in FileInfo :+1:

**Review by jsonsivar — COMMENTED** [review]
This looks good so far. It looks like the CI test broke so once we fix that and maybe do some testing like you said, we should be good to merge this.

File: core/space/domain/domain.go (hunk 1 above)
Comment (jsonsivar) [reply core/space/domain/domain.go:158]: No that's fine you can go ahead and remove it. But we should update sharing.go `GetReceivedFiles` to move members into the file info.  I searched across the repo and it seems like that is the only reference to it :+1:

**Review by jsonsivar — APPROVED** [review]
LGTM :+1: once we remove the extra `Members` field and then maybe test with grpc or swagger UI the nwe can go ahead and merge this.

**Review by dmerrill6 — APPROVED** [review]
LGTM

**Pull request merged**